servers/zts/src/main/java/com/yahoo/athenz/zts/utils/ZTSUtils.java

```diff
@@ -56,12 +58,16 @@
     public static final String ZTS_CERT_DNS_SUFFIX =
             System.getProperty(ZTSConsts.ZTS_PROP_CERT_DNS_SUFFIX, ZTSConsts.ZTS_CERT_DNS_SUFFIX);
 
-    private static final String ATHENZ_PROP_KEYSTORE_PASSWORD      = "athenz.ssl_key_store_password";
-    private static final String ATHENZ_PROP_TRUSTSTORE_PASSWORD    = "athenz.ssl_trust_store_password";
-    private static final String ATHENZ_PROP_KEYSTORE_PATH          = "athenz.ssl_key_store";
-    private static final String ATHENZ_PROP_KEYSTORE_TYPE          = "athenz.ssl_key_store_type";
-    private static final String ATHENZ_PROP_TRUSTSTORE_PATH        = "athenz.ssl_trust_store";
-    private static final String ATHENZ_PROP_TRUSTSTORE_TYPE        = "athenz.ssl_trust_store_type";
+    private static final String ATHENZ_PROP_KEYSTORE_PASSWORD                   = "athenz.ssl_key_store_password";
+    private static final String ATHENZ_PROP_TRUSTSTORE_PASSWORD                 = "athenz.ssl_trust_store_password";
+    private static final String ATHENZ_PROP_KEYSTORE_PATH                       = "athenz.ssl_key_store";
+    private static final String ATHENZ_PROP_KEYSTORE_TYPE                       = "athenz.ssl_key_store_type";
+    private static final String ATHENZ_PROP_TRUSTSTORE_PATH                     = "athenz.ssl_trust_store";
+    private static final String ATHENZ_PROP_TRUSTSTORE_TYPE                     = "athenz.ssl_trust_store_type";
+    private static final String ATHENZ_PROP_LOCAL_CERT_PATH_FOR_SSL             = "athenz.ssl_local_cert_path";
```

Comment:
instead of local in both places let's use public and indicate this is for client:

private static final String ATHENZ_PROP_PUBLIC_CERT_PATH_FOR_SSL = "athenz.ssl_client_public_cert_path";

Reply:
Fixed